proposals/stdlib/hex-format.md

```diff
@@ -306,7 +306,7 @@ Only a subset of Kotlin Standard Library available on all supported platforms is
 * Adding the ability to limit the number of hex digits when formatting numeric values
   * `NumberHexFormat.maxLength` could be introduced
     * When formatting an `Int`, combination of `maxLength = 6` and `removeLeadingZeros = false` results to exactly 6 least significant hex digits
-    * Combination of `maxLength = 6` and `removeLeadingZeros = true` returns maximum of 6 hex digits without leading zeros
+    * Combination of `maxLength = 6` and `removeLeadingZeros = true` returns a maximum of 6 hex digits without leading zeros
```

Comment:
Can be read as that one maximal digit among 6 digits is returned. Perhaps better "at most 6 hex (least-significant) digits"

Reply:
fixed